### PR TITLE
feat: Axelar spec from Impulse

### PR DESCRIPTION
**Pull request by MichaelLLC**
Hello, I'm Michael, founder of Impulse.expert, specializing in high-performance, secure, and scalable node operations across 21 chains, maintaining 99% uptime. I've been actively involved with Lava since October 21, 2022, contributing to docs, socials and automation scripts. I'm deeply engaged in community support and project development. The custom automation and monitoring systems ensure reliable operations. I'm eager to maintain the Axelar RPC spec on Lava and continue supporting the community with my expertise.

Commission: 2.5%:

    1% - Support and run a provider
    0.5% - Inherit, maintain and update Axelar spec
    1% - Provider support

Discord: michaels1473

### CODE REVIEW

File: mainnet-1/specs/axelar.json [hunk 1]
```diff
@@ -0,0 +1,2656 @@
...
+    "proposal": {
+        "title": "Add Specs: Axelar",
+        "description": "Adding new specification support for relaying axelar data on Lava",
+        "specs": [
+            {
+                "index": "AXELAR",
+                "name": "axelar mainnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDK50",
```
Comment (RuslanGlaznyov) [inline mainnet-1/specs/axelar.json:11]: Axelar uses Cosmos SDK v0.45. Perhaps we could use the Cosmos SDK spec with v0.47 already on chain if there are no API differences?

Comment (MichaelLLC) [reply mainnet-1/specs/axelar.json]: Updated the import parameter. Replaced the COSMOSSDK50 specification with COSMOSSDK

**Comment by MichaelLLC** [comment]
- These APIs do not work, so wasn't added to the spec:
`/axelar/axelarnet/v1beta1/chain_by_ibc_path/{ibc_path}`
`/axelar/axelarnet/v1beta1/ibc_path/{chain}`
- reviewed the imports, and it seems that COSMOSSDK and COSMOSWASM are sufficient, as COSMOSSDK47 includes COSMOSSDK -> ["IBC", "TENDERMINT"], also COSMOSSDK50 includes methods which currently unavailable in Axelar chain.

**Review by shleikes — CHANGES_REQUESTED** [review]

File: mainnet-1/specs/axelar.json [hunk 2]
```diff
@@ -0,0 +1,2656 @@
...
+        "description": "Adding new specification support for relaying axelar data on Lava",
+        "specs": [
+            {
+                "index": "AXELAR",
+                "name": "axelar mainnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDK",
+                    "COSMOSWASM"
+                ],
```
[the remark this answers is not in the document]
Comment (RuslanGlaznyov) [reply mainnet-1/specs/axelar.json:13]: @shleikes To clarify! We don't have COSMOSSDK45DEP COSMOSSDKFULL spec in mainnet

[the remark this answers is not in the document]
Comment (MichaelLLC) [reply mainnet-1/specs/axelar.json:13]: _Also, COSMOSSDKFULL is inheriting from COSMOSSDK and COSMOSWASM. Why not use that instead?_
There are several versions of the SDK available now. Using the COSMOSSDKFULL specification does not give me the flexibility to choose SDK versions.

[the remark this answers is not in the document]
Comment (shleikes) [reply mainnet-1/specs/axelar.json:13]: Got it

**Review by shleikes — APPROVED** [review]

**Review by ignacio-rsg — APPROVED** [review]

**Pull request merged**